cli/ballerina-cli/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -1,75 +1,60 @@
-       Compile Ballerina sources and produce the executable JAR file. However,
-       if the package does not contain an entry point, it will produce a non-executable
-       JAR file, which can be used in another package/program.
+       Compile the current package and its dependencies to produce a
+       standalone executable JAR (Java Archive) file.
 
-       Building a Ballerina package creates the executable `.jar` file
-       in the `<package-root>/target/bin` directory.
+       Build creates the resulting executable in the `target/bin` directory
+       when compiling the current package.
 
-       Building a standalone Ballerina file will create an executable `.jar` file in the
-       current directory. The name of the executable file will be
-       <ballerina-file-name>.jar'.
+       Build creates the resulting executable in the current directory
+       when compiling a single .bal file. The generated executable is named
+       after the source file.
 
-       Note: Building `.bal` files, which belong to a Ballerina package is not allowed.
+       Note: Building individual '.bal' files of the current package is not allowed.
 
 
 OPTIONS
-       -o, --output <output>
-           Write the output to the given file. The provided output
-           file name may or may not contain the ‘.jar’ extension. The -o flag will only work
-           for standalone Ballerina files.
+       --export-openapi
+           Generate openAPI definition for service declarations in the current package.
 
-       --offline
-            Build offline using the local artifacts without downloading the latest artifacts of
-            the dependencies from the remote repository (Ballerina Central).
+       --cloud <provider>
+           Generate cloud artifacts: '--cloud=k8s' for Kubernetes
+           and '--cloud=docker' for Docker.
 
-       --sticky
-       		Disable automatic updates.
-       		This will stick to the exact versions locked in the
-       		'Dependencies.toml' file (if exists).
+       --list-conflicted-classes
+           List the conflicting classes of conflicting JARs in the package.
 
-       --experimental
-           Enable experimental language features.
+       -o <output>, --output <output>
+           Write the output to the given file. These options will only work
+           for the single .bal file scenario.
 
        --observability-included
-           Include the dependencies and artifacts that are required to enable the observability functionalities
-           in the  executable JAR file.
+           Include the dependencies that are required to enable observability.
 
-       --cloud <provider>
-           Enable cloud artifact generation for cloud providers: (`--cloud=k8s`) for Kubernetes
-           and (`--cloud=docker`) for Docker.
+       --offline
+            Proceed without accessing the network. Attempt to proceed with the previously
+            downloaded dependencies in local caches fail otherwise.
```

Comment:
```suggestion
            downloaded dependencies in local caches, fail otherwise.
```

or

```suggestion
            downloaded dependencies in local caches or fail otherwise.
```

maybe?

Reply:
Fixed. I like the latter